lib/src/dynamics/island.dart

```diff
@@ -174,34 +149,34 @@ class Island {
     _listener = listener;
 
     if (_bodies == null || _bodyCapacity > _bodies.length) {
-      _bodies = new List<Body>(_bodyCapacity);
+      _bodies = List<Body>(_bodyCapacity);
     }
     if (_joints == null || _jointCapacity > _joints.length) {
-      _joints = new List<Joint>(_jointCapacity);
+      _joints = List<Joint>(_jointCapacity);
     }
     if (_contacts == null || _contactCapacity > _contacts.length) {
-      _contacts = new List<Contact>(_contactCapacity);
+      _contacts = List<Contact>(_contactCapacity);
     }
 
     // dynamic array
     if (_velocities == null || _bodyCapacity > _velocities.length) {
       final List<Velocity> old =
-          _velocities == null ? new List<Velocity>(0) : _velocities;
-      _velocities = new List<Velocity>(_bodyCapacity);
+          _velocities == null ? List<Velocity>(0) : _velocities;
+      _velocities = List<Velocity>(_bodyCapacity);
       BufferUtils.arraycopy(old, 0, _velocities, 0, old.length);
```

Comment:
These arraycopy is necessary?

Reply:
The buffers are filled with empty/placeholder items in the end of the lists as it is now, it would be nice to get away from but that would be a really big refactor since that behaviour is used everywhere in the code base.